msautotest/wxs/wfs_postgis_10f_precision.map

```diff
@@ -0,0 +1,71 @@
+#
+# Test rounding geometry floating point number to 10 decimal places support in PostGIS input and GML/WFS output
+#
+# REQUIRES: SUPPORTS=WFS INPUT=POSTGIS
+#
+#
+# RUN_PARMS: wfs_postgis_10f_precision_gml2.xml [MAPSERV] QUERY_STRING="map=[MAPFILE]&SERVICE=WFS&VERSION=1.0.0&REQUEST=GetFeature&TYPENAME=popplace" > [RESULT]
```

Comment:
you could possibly add &MAXFEATURES=1 in the requests to keep the size of the resulting files smaller (having more than 1 features doesn't add more to the testing)

Reply:
thanks for the remark. I modified the test in the last commit